src/Swift.Bindings/tests/IntegrationTests/FunctionalTests/Generics/GenericTests.swift

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import Foundation
+
+@frozen
+public struct FrozenStruct
+{
+    public var x: Int
+    public var y: Int
+
+    public init(x: Int, y: Int) {
+        self.x = x
+        self.y = y
+    }
+}
+
+public struct NonFrozenStruct
+{
+    public var x: Int
+    public var y: Int
+
+    public init(x: Int, y: Int) {
+        self.x = x
+        self.y = y
+    }
+}
+
+public func AcceptsGenericParametersAndThrows<T, U> (a: T, b: U) throws -> Int {
+    throw NSError()
```

Comment:
Why do we throw `NSError` instead of swift error? Is it simpler to catch at the currently?